helm/alfresco-content-services/templates/config-infrastructure.yaml

```diff
@@ -28,11 +28,11 @@ data:
     {{- else if eq "solr6" $search_flavor }}
       {{- $search_url = printf "http://%s/solr" (include "alfresco-search-service.deployment.name" .) }}
     {{- else if eq "elasticsearch" $search_flavor }}
+      {{- $esHost := printf "%s-%s" (.Release.Name | default "acs") (.Values.global.elasticsearch.service.name | default "elasticsearch") }}
+      {{- $esPort := .Values.global.elasticsearch.service.ports.restApi | default 9200 }}
```

Comment:
I'm not sure why this has been moved outside of the `if .enabled`.

Reply:
I'll move it back and use `$`